configs/components/openssl.rb

```diff
@@ -127,19 +128,22 @@
       #{target} \
       #{sslflags} \
       no-camellia \
-      enable-seed \
       enable-tlsext \
       enable-rfc3779 \
-      enable-cms \
       no-md2 \
       no-mdc2 \
-      no-rc5 \
```

Comment:
Pardon the delay in getting back. I did/do not have notifications enabled in github.. 
Yes weak-ssl-ciphers seems to disables it. Strangely enough it would not disable RC4 IIRC even though the documentation says it would be so need to add no-rc4 separately.